### PR TITLE
Add Main.java and MainTest.java

### PR DESCRIPTION
**Pull request by NicolasOgi**
Please accept. #8

### CODE REVIEW

File: src/test/java/ch/heig_vd/app/MainTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package ch.heig-vd.app;
+package ch.heig_vd.app;
```
Comment (MazieroMarco) [inline src/test/java/ch/heig_vd/app/MainTest.java:1]: Essential changes !

**Comment by LeonardBesseau** [comment]
Why did you delete the file ?

File: src/test/java/ch/heig_vd/app/MainTest.java (hunk 1 above)
Comment (LeonardBesseau) [reply src/test/java/ch/heig_vd/app/MainTest.java:1]: I concur

File: src/test/java/ch/heig_vd/app/MainTest.java [hunk 2]
```diff
@@ -17,4 +17,4 @@ public void shouldAnswerWithTrue()
     {
         assertTrue( true );
     }
-}
+}
```
Comment (MazieroMarco) [inline src/test/java/ch/heig_vd/app/MainTest.java:20]: Coded like a pro

**Review by LeonardBesseau — CHANGES_REQUESTED** [review]
Nothing to review

**Review by MazieroMarco — APPROVED** [review]
Overall it is a quite boring code but it does the job !

**Review by LeonardBesseau — APPROVED** [review]
It's good

File: src/test/java/ch/heig_vd/app/MainTest.java (hunk 1 above)
Comment (NicolasOgi) [reply src/test/java/ch/heig_vd/app/MainTest.java:1]: ok.

**Comment by LeonardBesseau** [comment]
UNACCEPTABLE
image

File: src/test/java/ch/heig_vd/app/MainTest.java (hunk 2 above)
Comment (NicolasOgi) [reply src/test/java/ch/heig_vd/app/MainTest.java:20]: > UNACCEPTABLE
> image

Oui

**Pull request merged**